content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -278,8 +278,6 @@ pod usage is still within acceptable limits.
 
 ### Container resource metrics
 
```

Comment:
We've a new way that's more data-driven
```suggestion

{{< feature-state feature_gate_name="HPAContainerMetrics" >}}
```
This will render the same as `{{< feature-state for_k8s_version="v1.30" state="stable" >}}`, but with an added tooltip.

Reply:
Cool